client/src/components/CardMaker.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import Card from '@material-ui/core/Card';
```

Comment:
Is there any way these material UI imports can be destructured to make this look a bit cleaner?